verification/src/transaction_verifier.rs

```diff
@@ -262,32 +195,6 @@ impl<'a> DuplicateDepsVerifier<'a> {
     }
 }
 
-pub struct NullVerifier<'a> {
```

Comment:
Why remove `NullVerifier` ?

Reply:
All resolved transactions contain live cell only now, so `NullVerifier` is unnecessary